python/dolfinx/wrappers/mesh.cpp

```diff
@@ -242,6 +242,15 @@ void mesh(py::module& m)
                                    self.values().size(), self.values().data(), \
                                    py::none());                                \
                              })                                                \
+      .def(                                                                    \
+          "find_indices",                                                      \
+          [](dolfinx::mesh::MeshTags<SCALAR>& self, SCALAR i) {                \
+            std::vector<std::int32_t> indices_i = self.find_indices(i);        \
+            return py::array_t<std::int32_t>(indices_i.size(),                 \
```

Comment:
Why is pybind11 not automatically converting the output of
```
.def("find_indices", &dolfinx::mesh::MeshTags<SCALAR>::find_indices)
```
to a list?

Reply:
It is and that is what really hurts. We do not want a list of ints here but rather a numpy array (because of all issues why iterating over large python's list is terrible idea, data locality etc)